services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/file/CloudFrontCache.java

```diff
@@ -0,0 +1,62 @@
+package org.sagebionetworks.repo.manager.file;
+
+import com.google.common.cache.CacheBuilder;
+import com.google.common.cache.CacheLoader;
+import com.google.common.cache.LoadingCache;
+import org.sagebionetworks.StackConfiguration;
+import org.sagebionetworks.repo.manager.KeyPairUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.security.PrivateKey;
+import java.util.concurrent.TimeUnit;
+
+
+public class CloudFrontCache {

```

Comment:
we should annotate this class with @Service annotation instead of manager-sbb.xml